sturmbahnfahrer-1.3-orig/bipedobject.h ve sturmbahnfahrer-1.3/bipedobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/brickwall.h ve sturmbahnfahrer-1.3/brickwall.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/carobject.cxx ve sturmbahnfahrer-1.3/carobject.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/carobject.h ve sturmbahnfahrer-1.3/carobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/carpettrack.h ve sturmbahnfahrer-1.3/carpettrack.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/cartobject.cxx ve sturmbahnfahrer-1.3/cartobject.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/cartobject.h ve sturmbahnfahrer-1.3/cartobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/controller.h ve sturmbahnfahrer-1.3/controller.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/controllerkey.cxx ve sturmbahnfahrer-1.3/controllerkey.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/controllerpad.cxx ve sturmbahnfahrer-1.3/controllerpad.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/cratewall.h ve sturmbahnfahrer-1.3/cratewall.h dosyaları birbirinin aynı
Ortak alt dizinler: sturmbahnfahrer-1.3-orig/debian ve sturmbahnfahrer-1.3/debian
sturmbahnfahrer-1.3-orig/doorstand.h ve sturmbahnfahrer-1.3/doorstand.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/dynamicboxobject.h ve sturmbahnfahrer-1.3/dynamicboxobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/dynamiccylobject.h ve sturmbahnfahrer-1.3/dynamiccylobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/dynamicobject.cxx ve sturmbahnfahrer-1.3/dynamicobject.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/dynamicobject.h ve sturmbahnfahrer-1.3/dynamicobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/ferriswheel.h ve sturmbahnfahrer-1.3/ferriswheel.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/INSTALL ve sturmbahnfahrer-1.3/INSTALL dosyaları birbirinin aynı
Ortak alt dizinler: sturmbahnfahrer-1.3-orig/images ve sturmbahnfahrer-1.3/images
sturmbahnfahrer-1.3-orig/intro.cxx ve sturmbahnfahrer-1.3/intro.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/intro.h ve sturmbahnfahrer-1.3/intro.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/joydb.h ve sturmbahnfahrer-1.3/joydb.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/JOYSTICKS ve sturmbahnfahrer-1.3/JOYSTICKS dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/LICENCE ve sturmbahnfahrer-1.3/LICENCE dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/main.cxx ve sturmbahnfahrer-1.3/main.cxx dosyaları birbirinin aynı
diff -Naus sturmbahnfahrer-1.3-orig/Makefile sturmbahnfahrer-1.3/Makefile
--- sturmbahnfahrer-1.3-orig/Makefile	2007-04-08 10:01:21.000000000 +0300
+++ sturmbahnfahrer-1.3/Makefile	2007-04-08 10:14:35.000000000 +0300
@@ -11,7 +11,7 @@
 
 OBJS=staticworldobject.o carobject.o controllerpad.o controllerkey.o main.o dynamicobject.o texture.o tga.o intro.o soundenginealsa.o vectortext.o postscore.o cartobject.o
 
-LIBS= $(ODEPREFIX)/lib/libode.a -lplibssgaux -lplibssg -lplibsg -lplibpu -lplibfnt -lplibul -lglut -lGLU -lGL -lasound
+LIBS= $(ODEPREFIX)/lib/libode.so -lplibssgaux -lplibssg -lplibsg -lplibpu -lplibfnt -lplibul -lglut -lGLU -lGL -lasound
 
 
 all: sturmbahnfahrer
sturmbahnfahrer-1.3-orig/modelmap.h ve sturmbahnfahrer-1.3/modelmap.h dosyaları birbirinin aynı
Ortak alt dizinler: sturmbahnfahrer-1.3-orig/models ve sturmbahnfahrer-1.3/models
sturmbahnfahrer-1.3-orig/plodegui.h ve sturmbahnfahrer-1.3/plodegui.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/postscore.cxx ve sturmbahnfahrer-1.3/postscore.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/postscore.h ve sturmbahnfahrer-1.3/postscore.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/README ve sturmbahnfahrer-1.3/README dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/resizebug.cxx ve sturmbahnfahrer-1.3/resizebug.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/soundclip.h ve sturmbahnfahrer-1.3/soundclip.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/soundenginealsa.cxx ve sturmbahnfahrer-1.3/soundenginealsa.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/soundenginealsa.h ve sturmbahnfahrer-1.3/soundenginealsa.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/soundfeed.h ve sturmbahnfahrer-1.3/soundfeed.h dosyaları birbirinin aynı
Ortak alt dizinler: sturmbahnfahrer-1.3-orig/sounds ve sturmbahnfahrer-1.3/sounds
sturmbahnfahrer-1.3-orig/spikejump.h ve sturmbahnfahrer-1.3/spikejump.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/starsky.h ve sturmbahnfahrer-1.3/starsky.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/staticworldobject.cxx ve sturmbahnfahrer-1.3/staticworldobject.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/staticworldobject.h ve sturmbahnfahrer-1.3/staticworldobject.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/stereocontext.h ve sturmbahnfahrer-1.3/stereocontext.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/sturmbahnfahrer.keys.example ve sturmbahnfahrer-1.3/sturmbahnfahrer.keys.example dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/sturmbahn.h ve sturmbahnfahrer-1.3/sturmbahn.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/texture.cxx ve sturmbahnfahrer-1.3/texture.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/texture.h ve sturmbahnfahrer-1.3/texture.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/tga.cxx ve sturmbahnfahrer-1.3/tga.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/tga.h ve sturmbahnfahrer-1.3/tga.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/TODO ve sturmbahnfahrer-1.3/TODO dosyaları birbirinin aynı
Ortak alt dizinler: sturmbahnfahrer-1.3-orig/tools ve sturmbahnfahrer-1.3/tools
sturmbahnfahrer-1.3-orig/trackingcam.h ve sturmbahnfahrer-1.3/trackingcam.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/usercam.h ve sturmbahnfahrer-1.3/usercam.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/vectortext.cxx ve sturmbahnfahrer-1.3/vectortext.cxx dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/vectortext.h ve sturmbahnfahrer-1.3/vectortext.h dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/workshop.txt ve sturmbahnfahrer-1.3/workshop.txt dosyaları birbirinin aynı
sturmbahnfahrer-1.3-orig/worldobject.h ve sturmbahnfahrer-1.3/worldobject.h dosyaları birbirinin aynı
